File: src/test/java/coreTDD/tennis/TennisMatchTest.java
package coreTDD.tennis;

import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class TennisMatchTest {

    @Test
    public void a_game_should_begin_love_love() {
        TennisMatch tennisMatch = new TennisMatch();

        assertThat(tennisMatch.getScore()).isEqualTo("Love-all");
    }
}

File: src/test/kotlin/kotlinCoreTDDTests/tennis/TennisMatchTest.kt
package kotlinCoreTDDTests.tennis

import kotlinCoreTDD.tennis.TennisMatch
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class TennisMatchTest {

    @Test
    fun theInitialScoreIs_Love_all() {
        assertThat(TennisMatch().getScore()).isEqualTo("Love-all")
    }
}